.github/workflows/push.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+on:
+  push:
+    branches: [main]
+jobs:
+  main:
+    name: Deploy to GitHub pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: w3c/spec-prod@v2
+        with:
+          GH_PAGES_BRANCH: gh-pages
+          TOOLCHAIN: bikeshed
+          SOURCE: index.bs
+          DESTINATION: index.html 
```

Comment:
The previous script separated outputs from sources and had out/index.html as the destination.

(Not sure that matters much, or whether there is an implied destination in the script.)